apps/alert_processor/test/alert_processor/rules_engine/notification_builder_test.exs

```diff
@@ -83,4 +85,84 @@ defmodule AlertProcessor.NotificationBuilderTest do
       assert %Notification{phone_number: ^expected_phone_number, email: nil} = notification
     end
   end
+
+  describe "Notification.call_to_action" do
+    setup do
+      user = insert(:user)
+      sub = insert(:subscription, user: user)
+
+      {:ok, user_subs: {user, [sub]}}
+    end
+
+    test "includes dotcom url and mbta go CTA if single bus route is effected", %{
+      user_subs: user_subs
+    } do
+      now = naive_to_local(~N[2018-01-11 14:10:55Z])
+      one_hour_ago = DateTime.add(now, -3600)
+      two_days_from_now = DateTime.add(now, 172_800)
+      three_days_from_now = DateTime.add(now, 259_200)
+
+      alert = %Alert{
+        id: "1",
+        header: nil,
+        effect_name: "Delay",
+        active_period: [%{start: two_days_from_now, end: three_days_from_now}],
+        informed_entities: [%InformedEntity{route_type: 3, route: "1"}],
+        last_push_notification: one_hour_ago
+      }
+
+      notification = NotificationBuilder.build_notification(user_subs, alert)
+
+      assert %Notification{
+               call_to_action:
+                 "Track your bus at https://mbta.com/route/1 or use the MBTA Go app: https://go.mbta.com"
+             } = notification
+    end
+
+    test "includes only mbta go CTA if multiple bus routes are effected", %{user_subs: user_subs} do
+      now = naive_to_local(~N[2018-01-11 14:10:55Z])
+      one_hour_ago = DateTime.add(now, -3600)
+      two_days_from_now = DateTime.add(now, 172_800)
+      three_days_from_now = DateTime.add(now, 259_200)
+
+      alert = %Alert{
+        id: "1",
+        header: nil,
+        effect_name: "Delay",
+        active_period: [%{start: two_days_from_now, end: three_days_from_now}],
+        informed_entities: [
+          %InformedEntity{route_type: 3, route: "1"},
+          %InformedEntity{route_type: 3, route: "66"},
+          %InformedEntity{route_type: 3, route: "72"}
+        ],
+        last_push_notification: one_hour_ago
+      }
+
+      notification = NotificationBuilder.build_notification(user_subs, alert)
+
+      assert %Notification{
+               call_to_action: "To track your bus, use the MBTA Go app: https://go.mbta.com"
+             } = notification
+    end
+
+    test "nil cta when not a delay alert", %{user_subs: user_subs} do
```

Comment:
I'd probably add a quick test making sure it behaves correctly for alerts where the effect name is "Delay" but the mode is something else- eg Commuter Rail delay alerts where the route_type is 2.